packages/lesswrong/server/callbacks/postCallbackFunctions.tsx

```diff
@@ -914,30 +915,33 @@ export async function updatedPostMaybeTriggerReview({newDocument, oldDocument, c
   }
 }
 
-export async function sendRejectionPM({ newDocument: post, oldDocument: oldPost, currentUser, context }: UpdateCallbackProperties<'Posts'>) {
+export async function sendRejectionPM({ post, currentUser, context }: {post: DbPost, currentUser?: DbUser|null, context: ResolverContext}) {
   const { Users } = context;
+  const postUser = await Users.findOne({_id: post.userId});
+
+  const rejectedContentLink = `<span>post, <a href="https://lesswrong.com/posts/${post._id}/${post.slug}">${post.title}</a></span>`
+
+  let messageContents = getRejectionMessage(rejectedContentLink, post.rejectedReason)
+
+  // FYI EA Forum: Decide if you want this to always send emails the way you do for deletion. We think it's better not to.
+  const noEmail = isEAForum
+  ? false 
+  : !(!!postUser?.reviewedByUserId && !postUser.snoozedUntilContentCount)
+  const adminAccount = currentUser ?? await getAdminTeamAccount(context);
```

Comment:
Has anything changed in this function when split up into `maybe` (apart from losing a newline here)?